iocBoot/iocLabJack/generate_T7_2_and_3_substitutions.sh

```diff
@@ -0,0 +1,6 @@
+cp LabJack_T7_1.substitutions LabJack_T7_2.substitutions
+cp LabJack_T7_1.substitutions LabJack_T7_3.substitutions
+sed -i s/:1:/:2:/ LabJack_T7_2.substitutions
+sed -i s/_1_/_2_/ LabJack_T7_2.substitutions
+sed -i s/:1:/:3:/ LabJack_T7_3.substitutions
+sed -i s/_1_/_4_/ LabJack_T7_3.substitutions
```

Comment:
Wouldn't this be s/_1_/_3_/ ?

Reply:
Thanks for catching that.  I had only tested LabJack_T7_2.substitutions, not 3.